### DISCUSSION TITLE
Flutter build Ios failed using internationalization (intl)

### DISCUSSION THREAD

**Discussion by printpagestopdf** [codemagic-ci-cd/codemagic-docs#2244] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

My flutter Project is using intl following this https://docs.flutter.dev/ui/accessibility-and-internationalization/internationalization walkthrough. Especially this part:
I have a file called l10n.yaml in projects root with this content:
arb-dir: lib/l10n
template-arb-file: app_en.arb
output-localization-file: app_localizations.dart

In dart code importing:
import 'package:flutter_gen/gen_l10n/app_localizations.dart';

The ios build process complains that it cannot find package flutter_gen:
Error (Xcode): Error: Couldn't resolve the package 'flutter_gen' in 'package:flutter_gen/gen_l10n/app_localizations.dart'.

My assumption is, that intl package is defining a synthetic package (by l10n.yaml) and the build process cannot resolve it. But I habe no idea how to solve it.

### Expected results

Build process should not fail because of not finding lutter_gen

### Actual results

The ios build process complains that it cannot find package flutter_gen:
Error (Xcode): Error: Couldn't resolve the package 'flutter_gen' in 'package:flutter_gen/gen_l10n/app_localizations.dart'.

### Build id (optional)

656f94790185bee877729835

**Comment by dtrdic** [comment, marked as answer]
Hello @printpagestopdf , can you please check out this [comment](https://github.com/flutter/flutter/issues/109606#issuecomment-1224972866). It does sound like a known issue being tracked [here](https://github.com/flutter/flutter/issues/84979), you can follow up on it for further updates

**Reply by printpagestopdf** [reply]
Sorry for my late reply but I was suffering from an infection. The comment you mentioned seemed to be the reason. I used the workaround mentioned there (moving synthetic package to real package). Not an elegant solution, but its working now.
Many thanks

**Reply by dtrdic** [reply]
Hope you have a speedy recovery, stay safe
